### PR TITLE
Update kommander to 1.45.0

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
This introduces new logout behavior and the chart updates allow
kommander to be configured as Konvoy UI.

### CODE REVIEW

File: stable/kommander/values.yaml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 image:
   repository: mesosphere/kommander
-  tag: 1.40.0
+  tag: 1.45.0
   pullPolicy: IfNotPresent
 replicas: 1
+isKonvoyUI: true
```
Comment (hectorj2f) [inline stable/kommander/values.yaml]: should not we default this to false ?

Comment (DanielMSchmidt) [reply stable/kommander/values.yaml]: yep, changed it, great catch 👍

File: stable/kommander/values.yaml [hunk 2]
```diff
@@ -1,8 +1,10 @@
 image:
   repository: mesosphere/kommander
-  tag: 1.40.0
+  tag: 1.45.0
   pullPolicy: IfNotPresent
 replicas: 1
+isKonvoyUI: false
+logoutRedirectPath: "/"
```
Comment (hectorj2f) [inline stable/kommander/values.yaml]: also, is this a default for kommander or it is only used for konvoy-ui too ?

Comment (hectorj2f) [reply stable/kommander/values.yaml]: @DanielMSchmidt ☝️

Comment (DanielMSchmidt) [reply stable/kommander/values.yaml]: I am unsure, it's the default we have in the backend code as well, so I think it's a save bet. @natmegs or @brandonc might know more about ops portal related logout routing

Comment (brandonc) [reply stable/kommander/values.yaml]: It's used in both

**Comment by shaneutt** [comment]
LGTM normally I would wait for the other reviewers, but we're in crunch time to start testing kommander for our release tomorrow so I'm going to merge so we can start testing asap.

**Pull request merged**